src/main.js

```diff
@@ -293,7 +293,7 @@ function createMenuItemForBoard() {
   menuItem.submenu.append(new MenuItem({ type: "separator" }));
   menuItem.submenu.append(
     new MenuItem({
-      label: "Export Using Board",
+      label: "Export Focused Board",
```

Comment:
フォーカスがあたっているボードをエクスポートするので。
とはいえこの文言は内部の仕様を知らない人に対して伝わるのだろうか…疑問。

Reply:
```suggestion
      label: "Export This Board",
```
でも通じる気がしますね。export対象のボードが全面に出ている状態に必ずなるので、thisが表示中のボードだってのは明確そう。(全画面で使ってる場合に限りそうですが　）